planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/src/utilities.cpp

```diff
@@ -43,7 +45,7 @@ rclcpp::Time safeAddition(const rclcpp::Time & t1, const double seconds)
 {
   rclcpp::Time sum;
   try {
-    sum = t1 + rclcpp::Duration::from_seconds(seconds);
+    sum = t1 + rclcpp::Duration(seconds);
```

Comment:
I guess it's better to use from_seconds function.

Reply:
Fixed.